components/sidebar/sidebar_channel/sidebar_channel_menu/sidebar_channel_menu.tsx

```diff
@@ -209,32 +209,16 @@ export class SidebarChannelMenu extends React.PureComponent<Props, State> {
 
         let leaveChannel;
         if (channel.name !== Constants.DEFAULT_CHANNEL) {
-            if (channel.type === Constants.DM_CHANNEL || channel.type === Constants.GM_CHANNEL) {
-                leaveChannel =
+            leaveChannel =
                     (<Menu.Group>
                         <Menu.ItemAction
                             id={`leave-${channel.id}`}
                             onClick={this.handleLeaveChannel}
                             icon={<i className='icon-close'/>}
                             text={leaveChannelText}
+                            isDangerous={(channel.type === Constants.DM_CHANNEL || channel.type === Constants.GM_CHANNEL) === false}
                         />
                     </Menu.Group>);
```

Comment:
and then we close it with the same indentation as before:

```suggestion
                    </Menu.Group>
                );
```